man/systemd.exec.xml

```diff
@@ -1727,13 +1741,13 @@ BindReadOnlyPaths=/var/lib/systemd</programlisting>
         <citerefentry><refentrytitle>sysctl.d</refentrytitle><manvolnum>5</manvolnum></citerefentry> mechanism. Few
         services need to write to these at runtime; it is hence recommended to turn this on for most services. For this
         setting the same restrictions regarding mount propagation and privileges apply as for
-        <varname>ReadOnlyPaths=</varname> and related calls, see above. Defaults to off.  If turned on and if running
-        in user mode, or in system mode, but without the <constant>CAP_SYS_ADMIN</constant> capability (e.g.  services
-        for which <varname>User=</varname> is set), <varname>NoNewPrivileges=yes</varname> is implied. Note that this
-        option does not prevent indirect changes to kernel tunables effected by IPC calls to other processes. However,
-        <varname>InaccessiblePaths=</varname> may be used to make relevant IPC file system objects inaccessible. If
-        <varname>ProtectKernelTunables=</varname> is set, <varname>MountAPIVFS=yes</varname> is
-        implied.</para>
+        <varname>ReadOnlyPaths=</varname> and related calls, see above. Defaults to off. If without the
+        <constant>CAP_SYS_ADMIN</constant> capability (e.g. setting <varname>User=</varname>),
+        <varname>NoNewPrivileges=yes</varname> is implied. Note that this option does not prevent
```

Comment:
The old sentence was:
`If turned on and if running in user mode, or in system mode, but without the constant>CAP_SYS_ADMIN</constant> capability (e.g.  services for which <varname>User=</varname> is set), <varname>NoNewPrivileges=yes</varname> is implied.`
The new setence is:
`If without the <constant>CAP_SYS_ADMIN</constant> capability (e.g. setting <varname>User=</varname>), <varname>NoNewPrivileges=yes</varname> is implied. `

Can we keep the original sentence structure, and only adjust it as minimally necessary? The old version was much easier to understand because it used simple concepts like "user mode" rather than the underlying technical detail "CAP_SYS_ADMIN". In the new version, the subject is only implied, and it's not obviously clear what it is.

Reply:
"running in user mode" means `pid != 1`, no? I dropped that part because the setting is only available for PID1.

Reply:
Indeed. So maybe something like this:
`If this setting is on, but the unit doesn't have the <constant>CAP_SYS_ADMIN</constant> capability (e.g.  services for which <varname>User=</varname> is set), <varname>NoNewPrivileges=yes</varname> is implied.`